docs/dev/system_keyspace.md

```diff
@@ -356,6 +356,12 @@ CREATE TABLE system.clients (
 Currently only CQL clients are tracked. The table used to be present on disk (in data
 directory) before and including version 4.5.
 
+# Regular tables in the system keyspace
+
+The virtual tables are volatile, and they are not persisted to disk. The data which
+are supposed to be durable across reboots are persisted as regular tables.
+They are listed below in alphabetical order.
+
```

Comment:
We already have this section, it is above the virtual tables one. The tables below just have to be moved over there.

Reply:
i see. the title reads "[System keyspace layout](https://github.com/scylladb/scylladb/blob/c67be070aa1dcfd871d01a475c3ed5305cf20480/docs/dev/system_keyspace.md#system-keyspace-layout)", so i thought it outlines the general layout of *all* tables in system keyspace.